doc/howtos/bluetoothspeaker.rst

```diff
@@ -0,0 +1,136 @@
+Bluetooth Speaker
+#################
+
+How to setup your intel-iot-refkit to function as a bluetooth speaker
+=====================================================================
+
+For bluetooth speaker setup you should use the "gateway" image and
+prepare couple of configuration files. This configuration has been
+tested to work with Intel Joule development board which has bluetooth
+radio chip integrated. Following 5 steps should make it:
+
+1. Start pulseaudio in the boot
+2. Add D-Bus configuration so that pulseaudio can communicate with BlueZ
+3. Configure BlueZ to enable A2DP sink
+4. Unblock bluetooth at boot with Connman
+5. Pair and connect your source device
+
+Start pulseaudio in the boot
+============================
+
+The way to start pulseuadio in the boot is to use systemd service
+file. In this example service file we start pulseaudio in system mode
+and don't let the daemon exit when in idle (which is normal
+pulseuaudio behavior). Pulseaudio will just go to "sleep"::
+
+  pulseaudio
+  [Unit]
+  Description=Pulse Audio
+
+  [Service]
+  Type=simple
+  ExecStart=/usr/bin/pulseaudio --system --disallow-exit --disable-shm --exit-idle-time=-1
+
+  [Install]
+  WantedBy=multi-user.target
+
+Copy the pulseaudio.service file to::
+
+  /etc/systemd/system/
+
+and enable it by::
+
+  systemctl enable pulseaudio.service
```

Comment:
yes they are. we don't have very good guides or recommendations how you should do this. Also the former appfw is not there to force you to create users. IMHO in this demo it doesn't too much matter even though we are running as root, it is basically a box with no wlan connectivity. If you get the device into your hands, you can pair the bluetooth with a button. I have the same demo working with a user also, but it includes also some "experimental" setup like configuring bluez and connman system dbus to be accessible from user and changing the group rights of sysfs gpio. So it might be better way to do it, but in my opinion doesn't bring in too much for security (in this use case).